packages/components-angular/src/app/examples/select-example-dynamic.component.ts

```diff
@@ -0,0 +1,61 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import type { SelectUpdateEventDetail } from '@porsche-design-system/components-angular';
+
+@Component({
+  selector: 'page-select-example-dynamic',
+  template: `
+    <label>
+      Value:
+      <input name="input-value" type="text" [value]="inputValue" (input)="onChangeInput($event)" placeholder="e.g. 1" />
+    </label>
+    <button type="button" (click)="onSetValue()">Set Value</button>
+    <button type="button" (click)="onResetValue()">Reset value</button>
```

Comment:
Namings across newly added (but probably also old) examples is mixed. Sometimes the handlers are called `handleSubmit`, but here it is essentially `onSubmit`.  
Consistency would be good.  
Same situation in react and I guess also vue.